docs/src/release_notes/21.8.0.rst

```diff
@@ -0,0 +1,102 @@
+21.8.0
+======
+
+Direct runner on GKE
+--------------------
+
+Klio can now be run on the additional runner of DirectGKERunner.
```

Comment:
Do we have any docs in this repo on running with direct runner? Can we link to them? If we don't have them, should we add some (not in this PR)?

Reply:
AH I just saw you created tasks for this - nvm!

Reply:
I just merged in #211 - is it too late to link it here? 🙃

Reply:
Just linked it!

Reply:
Tho I think the docs build will fail on the link since its in develop and not master?